public/app/plugins/panel/trend/TrendPanel.tsx

```diff
@@ -61,7 +60,7 @@ export const TrendPanel = ({
     } else {
       // first number field
       // Perhaps we can/should support any ordinal rather than an error here
-      xFieldIdx = frames[0].fields.findIndex((f) => f.type === FieldType.number);
+      xFieldIdx = frames[0] ? frames[0].fields.findIndex((f) => f.type === FieldType.number) : -1;
```

Comment:
I don't see any harm in adding this condition, but I'm worried that the problem can be in some different place. I mean why have empty frames in the first place?

Reply:
@Develer bro. do you think this pr can merge or not? or you have some suggestion. i will be pleasure to modified it.